Composer/packages/client/src/recoilModel/dispatchers/utils/project.ts

```diff
@@ -912,7 +912,6 @@ export const checkIfBotExistsInBotProjectFile = async (
 };
 
 export const getMemoryVariables = async (projectId: string, options?: { signal: AbortSignal }) => {
-  const res = await fetch(`${BASEURL}/projects/${projectId}/variables`, { signal: options?.signal });
-  const json = (await res.json()) as { variables: string[] };
-  return json.variables ?? [];
+  const res = await httpClient.get(`/projects/${projectId}/variables`, { signal: options?.signal });
```

Comment:
The BASEURL variable is no longer needed?

Reply:
No, the content of BASEURL is '/api', but the httpClient add it by default 
![image](https://github.com/southworks/BotFramework-Composer/assets/122501764/5129e1b1-39b4-4ec9-91d4-bc6ce7aa9f00)